.github/actions/spell-check/expect.txt

```diff
@@ -459,9 +360,9 @@ drawingcolor
 dreamsofameaningfullife
 drf
 drivedetectionwarning
-dsc
 dshow
 DSTINVERT
+ductdo
```

Comment:
This is a username. Why are we removing from names.txt and adding in expect.txt?